pxl_scripts/px/redis_data/manifest.yaml

```diff
@@ -0,0 +1,6 @@
+---
+short: Redis RPC messages
```

Comment:
"long: Shows a sample of Redis messages in the cluster." would better match the descriptions of the other `px/*_data` scripts.